apps/designer/app/designer/features/sidebar-left/panels/image-upload.tsx

```diff
@@ -1,37 +1,29 @@
 import { ImageIcon } from "~/shared/icons";
```

Comment:
image-upload.tsx should probabbly be asset-manager.tsx

Reply:
Done!